packages/server/src/tests/banners/ChannelBannerTests.ts

```diff
@@ -18,8 +13,6 @@ import {
     OphanProduct,
     RawTestParams,
     RawVariantParams,
-    TickerCountType,
-    TickerEndType,
```

Comment:
Do we only use tickers in these specialised, bespoke banners? I feel a bit nervous about deleting ticker-related code as we may have to add it back in again for the next bespoke banner?

Reply:
tickers are configurable this way from the epic tool, but not the banner tool. This is because we only ever use the ticker on moment banners, which have tended to have custom design. But it may be useful in future to support configuration from the banner tool